plugins/search/src/alpha.tsx

```diff
@@ -154,6 +155,11 @@ export const SearchPage = createPageExtension({
                         name: 'Architecture Decision Records',
                         icon: <DocsIcon />,
                       },
+                      {
```

Comment:
I'm feeling the need for filters to be extensions so I can install specific filters for Stack Overflow, any ideas?

Reply:
Thanks for sharing your thoughts!
It was removed for now, I'll convert into filters in a separate pull request 🙂